client/src/helpers/loginHelpers.js

```diff
@@ -0,0 +1,23 @@
+// here I'll fetch all my routesto the frontend with axios
```

Comment:
Well, maybe, apiCalls.js would have been a slightly better name for the file, because you do even more stuff than login here :)

Reply:
Yeah, I'll apply that name, thanks, sometimes is even harder to think about names for functions, variables or files then the code its self 😅